submissions/NastjonkaK/JS-DOM/index.html

```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
+<html lang="uk">
+
+<head>
+	<title>DOM</title>
+	<meta charset="UTF-8" />
+	<link rel="stylesheet" href="Style/style.css" />
+	<meta name="viewport" content="width=device-width, initial-scale=1.0" />
+</head>
+
+<body>
+	<div class="container">
+		<div class="content">
+			<div class="menu">
+				<ul class="countries">
+					<li id = "1">Австрія</li>
+					<li id = "2">Італія</li>
+					<li id = "3">Нідерланди</li>
+					<li id = "4">Чехія</li>
+					<li id = "5">Хорватія</li>
+				</ul>
+			</div>
+			<div class="description">
+				<h3 class="countryName"></h3>
+				<p class="shortDescription"></p>
+			</div>
+		</div>
+	</div>
+	<script src="script.js"></script>
+</body>
+
+</html>
```

Comment:
Add an empty line at the end of each file. [Reason](https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline?noredirect=1&lq=1)